extensions/smallrye-health/runtime/pom.xml

```diff
@@ -18,12 +18,6 @@
         <dependency>
             <groupId>io.smallrye</groupId>
             <artifactId>smallrye-health</artifactId>
-            <exclusions>
```

Comment:
If the exclusion is no longer needed, does that mean that `smallrye-health` now ships with `jakarta.inject`? If so, then the dependency below should be removed

Reply:
> If the exclusion is no longer needed, does that mean that smallrye-health now ships with jakarta.inject?

Yes it is.

> If so, then the dependency below should be removed

You're right! Thanks

Reply:
Excellent, thanks!